rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/vendor/VendorController.java

```diff
@@ -274,6 +274,56 @@ public ResponseEntity<?> exportVendor(HttpServletResponse response) throws TExce
         return ResponseEntity.ok(HttpStatus.OK);
     }
 
+    @PreAuthorize("hasAuthority('WRITE')")
+    @Operation(
+            summary = "Merge two vendors.",
+            responses = {
+                    @ApiResponse(responseCode = "200", description = "Merge successful.",
+                            content = {
+                                    @Content(mediaType = "application/json",
+                                            examples = @ExampleObject(
+                                                    value = "{\"message\": \"Merge successful.\"}"
+                                            ))
+                            }),
+                    @ApiResponse(responseCode = "400", description = "Merge failed.",
+                            content = {
+                                    @Content(mediaType = "application/json",
+                                            examples = @ExampleObject(
+                                                    value = "{\"message\": \"Merge failed.\"}"
```

Comment:
Please use actual message if possible.
```suggestion
                                                    value = "{\"message\": \"Vendor used as source or target has an open MR\"}"
```